sdcm/tester.py

```diff
@@ -1398,22 +1398,22 @@ def tearDown(self):
 
     def zip_and_upload_job_log(self):
 
-        event_log_dir_tmp = os.path.join(self.logdir, 'job_log')
```

Comment:
this I've also change in #913, but we didn't merged it yet

Reply:
OK, I will drop this patch.